pybadge.wpr

```diff
@@ -0,0 +1,13 @@
+#!wing
```

Comment:
This looks like a Wing IDE project file - is this something we we should be keeping in the main repo?